src/classes/ADDR_Seasonal_UTIL.cls

```diff
@@ -0,0 +1,259 @@
+/*
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.